iotqatools/simulators/listen/listen.js

```diff
@@ -58,26 +99,37 @@ process.argv.forEach(function (val, index, array) {
                 console.log("# Accumulator server version: " + version );
                 process.exit(0);
         }
-        if (val=="-u"){
+        if (val=="-u" || val=="--help" ){
```

Comment:
Is there any documentation piece associated to the scriot (i.e. a .md file)? Just in case the change in CLI needs to be sync in some other part...